external/elasticsearch/archetype/src/main/resources/archetype-resources/crawler-conf.yaml

```diff
@@ -46,6 +46,9 @@ config:
   http.protocol.implementation: "com.digitalpebble.stormcrawler.protocol.okhttp.HttpProtocol"
   https.protocol.implementation: "com.digitalpebble.stormcrawler.protocol.okhttp.HttpProtocol"
 
+  # Please define this for selenium grid configuration
+  selenium.grid.address: http://localhost:4444
```

Comment:
As the change is in `core`, the related adjustment of `crawler-conf.yaml` should also be included in the other archetypes.